### PR TITLE
Updating Wiki Stuff...

### PR DESCRIPTION
**Pull request by xamionex**
You can check this [here](https://amione.gitbook.io/northstarwiki)

### CODE REVIEW

File: docs/hosting-a-server-with-northstar/dedicated-server/best-practices.md [hunk 1]
```diff
@@ -17,14 +16,12 @@ The current minimum requirements are as follows:
 
 - 5GB free disk space (+3 GB if you're on windows due to origin and deps)
 - 3+ cores (2 might work, though)
-- 6GB+ total memory (RAM or swap will do) 
-- Windows 8.1 (and built-in WARP [use `-softwared3d11`] or a graphics card) + or wine 6.0.0+ (and dxvk+lavapipe+x11, wined3d+llvmpipe+x11, or a working graphics setup with dxvk or llvmpipe) (see https://git.io/JyrOR for wine details)
-
+- 6GB+ total memory (RAM or swap will do)
+- Windows 8.1 (and built-in WARP [use `-softwared3d11`] or a graphics card) + or wine 6.0.0+ (and dxvk+lavapipe+x11, wined3d+llvmpipe+x11, or a working graphics setup with dxvk or llvmpipe) (see [this](https://git.io/JyrOR) for more)
```
Comment (GeckoEidechse) [inline docs/hosting-a-server-with-northstar/dedicated-server/best-practices.md]: We should probably just unshorten the link by replacing `https://git.io/JyrOR` with `https://github.com/pg9182/northstar-dedicated`.

File: docs/modding/keyvalues.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+Do keep in note that this is usually changed server-side.
+
+To actually change the keyvalues of weapons you'd have to name them appropriately inside a mod folder\
+For example: `R2Northstar\mods\Northstar.Custom\keyvalues\scripts\weapons`\
+You can find all the weapon keyvalues [here](https://github.com/BigSpice/TitanFall_2_Weapon_Skin_Modding/tree/main/Weapon_Scripts)
+
+- Mod Examples:
+  - [Northstar.Custom](https://github.com/R2Northstar/NorthstarMods/tree/main/Northstar.Custom)
+  - [Unholy Trinity](https://github.com/xamionex/xamionex.UnholyTrinity)
+  - [Kraber9k](https://github.com/Steveplays28/kraber9k/tree/main/Steveplays.Kraber9K)
```
Comment (GeckoEidechse) [inline docs/modding/keyvalues.md:12]: Seems to be missing in `SUMMARY.md`.

File: docs/modding/tools.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+* [taskinoz RSPNVPK](https://github.com/taskinoz/RSPNVPK)
+* [Provoxin RSPNVPK](https://github.com/Provoxin/RSPNVPK-GUI)
+
+{% embed url="https://noskill.gitbook.io/titanfall2/how-to-start-modding/modding-introduction/modding-tools/how-to-use-rspnvpk" %}
+NoSkill: How to use RSPNVPKs
+{% endembed %}
+
+### VTF & VMT
+
+* [VTFEdit](this-content-is-no-longer-available.md)
```
Comment (GeckoEidechse) [inline docs/modding/tools.md]: I guess the link broke while editing? ^^

File: docs/modding/tools.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+## 3D Models
+
+* [Blender](https://www.blender.org)
+  * [Blender Source Tool](https://developer.valvesoftware.com/wiki/Blender\_Source\_Tools)
+  * [Blender VMT](https://github.com/lasa01/blender-vmt)
+  * [io import VMF & VMT](https://github.com/lasa01/io\_import\_vmf)
+* [Crowbar](http://steamcommunity.com/groups/CrowbarTool)
+* [Ninja Ripper v1.7.1](https://gamebanana.com/tools/5638)
+
+Highly recommended to check the [NoSkill gitbook](https://noskill.gitbook.io/titanfall2/how-to-start-modding/modding-introduction/modding-tools) for more tools
```
Comment (GeckoEidechse) [inline docs/modding/tools.md]:
```suggestion
Highly recommended to check the [NoSkill Gitbook modding section](https://noskill.gitbook.io/titanfall2/how-to-start-modding/modding-introduction/modding-tools) for more tools.
```

File: docs/modding/tools.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+  * [VTF plug-in](https://www.tophattwaffle.com/downloads/gimp-vtf-plugin/)
+* [RGB Tool converter](http://hewmc.blogspot.fr/2012/12/rgb-to-percentage-converter.html)
+* [Color Pick Windows](https://pixelbyte.itch.io/color-pick)
+* [GIF splitter](https://ezgif.com/split)
+* [Rad Tools Bik](http://www.radgametools.com/bnkdown.htm) (needs [Quicktime 7.6](https://support.apple.com/downloads/quicktime))
+
+### Archives
+
+* [7zip](https://www.7-zip.org)
+* [WinRAR](https://www.win-rar.com/download.html?L=10)
```
Comment (GeckoEidechse) [inline docs/modding/tools.md]: Side note, is there any reason to still use WinRAR? Considering that 7zip can even open `.rar` archives nowadays? ^^

File: docs/SUMMARY.md [hunk 6]
```diff
@@ -33,6 +33,7 @@
 ## Modding and Development
 
 * [Modding](modding/README.md)
+  * [Modding Tools](modding/tools.md)
```
Comment (GeckoEidechse) [inline docs/SUMMARY.md]: Probably best to move this section further down, at least below "Getting Started".

File: CODE_OF_CONDUCT.md [hunk 7]
```diff
@@ -59,8 +59,7 @@ representative at an online or offline event.
 ## Enforcement
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported to the community leaders responsible for enforcement at
-https://github.com/R2Northstar.
+reported to the community leaders responsible for enforcement [here](https://github.com/R2Northstar).
```
Comment (GeckoEidechse) [inline CODE_OF_CONDUCT.md:62]: `CODE_OF_CONDUCT.md` was based on a template. I'd just leave as is tbh.

File: docs/modding/README.md [hunk 8]
```diff
@@ -2,13 +2,14 @@
...
-* List of squirrel funcs available to dedicated server with their signatures - [https://gist.github.com/laundmo/a6ee0692ae137876b7514c0f4a57df4b](https://gist.github.com/laundmo/a6ee0692ae137876b7514c0f4a57df4b)
-* Native registered script functions, they are from Season 3 Apex but most also exist on Titanfall 2, especially the Titan functions. Also includes description for each function if the developers have given it one - [https://github.com/kawainekome/ApexLegends_S3_Squirrel_Registered_API_SimpleDoc](https://github.com/kawainekome/ApexLegends_S3_Squirrel_Registered_API_SimpleDoc)
+* [NoSkill modding guide (highly recommended to check out)](https://noskill.gitbook.io/titanfall2/)
+* [Tools for modding](tools.md)
+* [All the Squirrel Constants](https://gist.github.com/laundmo/825c4708663a179b23e144402b09244f)
+* [Modding Documentation](https://github.com/ScureX/Titanfall2-ModdingDocumentation#readme)
+* [Squirrel Highlighting for Npad++ and VSCode](https://gist.github.com/samisalreadytaken/5bcf322332074f31545ccb6651b88f2d)
```
Comment (GeckoEidechse) [inline docs/modding/README.md]:
```suggestion
* [Squirrel Highlighting for Notepad++ and VSCode](https://gist.github.com/samisalreadytaken/5bcf322332074f31545ccb6651b88f2d)
```

**Review by GeckoEidechse — CHANGES_REQUESTED** [review]
Looks good so far, thanks for the work <3

Left a few comments sprinkled throughout with things that should probably be changed.

Also what's with 
- `.github/tools/Titanfall_VPKTool3.4_Portable.zip`
- `.github/tools/vtfedit133.zip`
?

They are not linked to anywhere in the end anyway and I'm not sure what the LICENSE for redistribution for those is. So it would probably be better to remove them and link to their source. ^^

File: docs/modding/keyvalues.md (hunk 2 above)
Comment (xamionex) [reply docs/modding/keyvalues.md:12]: I didn't mean to add keyvalues but I had spare time, but i felt as though it should be expanded before I add it into SUMMARY

File: docs/modding/tools.md (hunk 3 above)
Comment (xamionex) [reply docs/modding/tools.md]: That is why it's present in .github/tools/ but might see if there's an archive.

File: CODE_OF_CONDUCT.md (hunk 7 above)
Comment (xamionex) [reply CODE_OF_CONDUCT.md:62]: Markdoown Ruuuleeees

Comment (GeckoEidechse) [reply CODE_OF_CONDUCT.md:62]: Ugh, fine

**Review by GeckoEidechse — APPROVED** [review]

**Pull request merged**